### ISSUE TITLE
How to build an engine file using the TensorRT API from a QDQ-onnx from pytorch_quantization QAT

### ISSUE THREAD

**Issue by songkq** [NVIDIA/TensorRT#2629]
@ttyio @zerollzeng Hi,  Could you please provide a tutorial about how to build an engine file using the TensorRT API from a QDQ-onnx/pth from pytorch_quantization QAT? Thanks~

**Comment by nvpohanh** [comment]
just load the ONNX file into TRT?

**Comment by songkq** [comment]
> just load the ONNX file into TRT?
@nvpohanh Yeah, I have tried trtexec but it failed（https://github.com/NVIDIA/TensorRT/issues/2616）. So I'm wondering whether I can build an engine through TensorRT API? Or is there anyway else to do it?

**Comment by nvpohanh** [comment]
Yes, you can use TRT API to construct the networks. The docs are here: https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html#c_topics
